test/e2e/support/setup-test-framework.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import 'expect-puppeteer';
+
+/**
+ * Internal dependencies
+ */
+import {
+	clearLocalStorage,
+	disablePageDialogAccept,
+	enablePageDialogAccept,
+	setViewport,
+	visitAdmin,
+} from './utils';
+
+/**
+ * Environment variables
+ */
+const { PUPPETEER_TIMEOUT } = process.env;
+
+// The Jest timeout is increased because these tests are a bit slow
+jest.setTimeout( PUPPETEER_TIMEOUT || 100000 );
+
+// Before every test suite run, delete all content created by the test. This ensures
+// other posts/comments/etc. aren't dirtying tests and tests don't depend on
+// each other's side-effects.
+beforeAll( async () => {
+	enablePageDialogAccept();
+
+	// Visit `/wp-admin/edit.php` so we can see a list of posts and delete them.
+	await visitAdmin( 'edit.php' );
```

Comment:
@tofumatt - so we now do cleanup before every test suite starts.

Reply:
This seems good to me.